tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "buildOnSave": false,
   "compilerOptions": {
     "module": "commonjs",
```

Comment:
I seem to recall @bennett000 talking about tree-shaking only working properly with `"module": "es6"`. It would be nice to roll that in here too.